CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+# 5.4.3 / 06-19-2015
+**Linux or Source Install only**
+
+### Details
+https://github.com/DataDog/dd-agent/compare/5.4.2...5.4.3
+
+### Changes
+* [SECURITY] The deb and rpm packages now bundle OpenSSL 1.0.1p
```

Comment:
Can you add a link to the security advisory ? http://openssl.org/news/secadv_20150709.txt